### PR TITLE
Feature/DTGB-147: Updated font sizes and paddings for buttons.

### PR DESCRIPTION
**Pull request by meirege**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.

### CODE REVIEW

File: styleguide/components/00-settings/_vars.scss [hunk 1]
```diff
@@ -87,32 +105,109 @@ $bp-desktop: 960px !default;
...
+///
+/// @since 3.0.0
+/// @group settings
+/// @access public
+/// @author Gert-Jan Meire
+///
 $default-font-size: 16px !default; //px
+
+///
+/// Defines the default line heighr.
```
Comment (Jeroen005) [inline styleguide/components/00-settings/_vars.scss]: typo

File: styleguide/components/11-base/fonts/_fonts.scss [hunk 2]
```diff
@@ -1 +1,5 @@
+////
+/// This file defines fonts.
+////
+
 @import url('https://fonts.googleapis.com/css?family=Fira+Sans'); // sass-lint:disable-line no-url-protocols
```
Comment (Jeroen005) [inline styleguide/components/11-base/fonts/_fonts.scss]: Maybe only include used font-weights and variants.

File: styleguide/components/21-atoms/buttons/button-alert/_button-alert.scss [hunk 3]
```diff
@@ -14,6 +14,10 @@
   color: color('white');
   box-shadow: $box-shadow-primary color('red', -3);
 
+  &.button-block {
+    text-align:  center;
```
Comment (Jeroen005) [inline styleguide/components/21-atoms/buttons/button-alert/_button-alert.scss]: space

File: styleguide/components/51-templates/buttons/buttons.twig [hunk 4]
```diff
@@ -39,20 +39,20 @@
 </div>
 <div class="preview-inner-inner-container">
   <p>
-    Small:
-    {% include '@button-secondary' with { 'modifier': 'button-small' } %}
-    Medium:
-    {% include '@button-secondary' with { 'modifier': 'button-medium' } %}
-    Large:
-    {% include '@button-secondary' with { 'modifier': 'button-large' } %}
+  {% include '@button-secondary' with { 'text': 'Small', 'modifier': 'button-small' } %}
```
Comment (Jeroen005) [inline styleguide/components/51-templates/buttons/buttons.twig]: indent

File: styleguide/components/51-templates/buttons/buttons.twig [hunk 5]
```diff
@@ -67,19 +67,19 @@
 </div>
 <div class="preview-inner-inner-container">
   <p>
-    Small:
-    {% include '@button-dark' with { 'modifier': 'button-small' } %}
-    Medium:
-    {% include '@button-dark' with { 'modifier': 'button-medium' } %}
-    Large:
-    {% include '@button-dark' with { 'modifier': 'button-large' } %}
+  {% include '@button-dark' with { 'text': 'Small', 'modifier': 'button-small' } %}
```
Comment (Jeroen005) [inline styleguide/components/51-templates/buttons/buttons.twig]: indent

File: styleguide/components/51-templates/buttons/buttons.twig [hunk 6]
```diff
@@ -95,19 +95,19 @@
 </div>
 <div class="preview-inner-inner-container">
   <p>
-    Small:
-    {% include '@button-alert' with { 'modifier': 'button-small' } %}
-    Medium:
-    {% include '@button-alert' with { 'modifier': 'button-medium' } %}
-    Large:
-    {% include '@button-alert' with { 'modifier': 'button-large' } %}
+  {% include '@button-alert' with { 'text': 'Small', 'modifier': 'button-small' } %}
```
Comment (Jeroen005) [inline styleguide/components/51-templates/buttons/buttons.twig]: indent

File: styleguide/components/51-templates/buttons/buttons.twig [hunk 7]
```diff
@@ -123,19 +123,19 @@
 </div>
 <div class="preview-inner-inner-container">
   <p>
-    Small:
-    {% include '@button-success' with { 'modifier': 'button-small' } %}
-    Medium:
-    {% include '@button-success' with { 'modifier': 'button-medium' } %}
-    Large:
-    {% include '@button-success' with { 'modifier': 'button-large' } %}
+  {% include '@button-success' with { 'text': 'Small', 'modifier': 'button-small' } %}
```
Comment (Jeroen005) [inline styleguide/components/51-templates/buttons/buttons.twig]: indent

**Review by Jeroen005 — APPROVED** [review]

File: styleguide/components/00-settings/_vars.scss (hunk 1 above)
Comment (meirege) [reply styleguide/components/00-settings/_vars.scss]: Fixed.

File: styleguide/components/51-templates/buttons/buttons.twig (hunk 7 above)
Comment (meirege) [reply styleguide/components/51-templates/buttons/buttons.twig]: Fixed.

File: styleguide/components/51-templates/buttons/buttons.twig (hunk 6 above)
Comment (meirege) [reply styleguide/components/51-templates/buttons/buttons.twig]: Fixed.

File: styleguide/components/51-templates/buttons/buttons.twig (hunk 5 above)
Comment (meirege) [reply styleguide/components/51-templates/buttons/buttons.twig]: Fixed.

File: styleguide/components/51-templates/buttons/buttons.twig (hunk 4 above)
Comment (meirege) [reply styleguide/components/51-templates/buttons/buttons.twig]: Fixed.

File: styleguide/components/21-atoms/buttons/button-alert/_button-alert.scss (hunk 3 above)
Comment (meirege) [reply styleguide/components/21-atoms/buttons/button-alert/_button-alert.scss]: Fixed.

File: styleguide/components/11-base/fonts/_fonts.scss (hunk 2 above)
Comment (meirege) [reply styleguide/components/11-base/fonts/_fonts.scss]: Added.

**Pull request merged**